apps/landingPage/src/components/Headline.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import React from 'react';
+import { Heading, Text, Button } from '@kiwicom/orbit-components/lib/';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import styled from 'styled-components';
+
+import kiwiLogo from '../images/logo.png';
+import reactLogo from '../images/react-logo.png';
+import tequilaLogo from '../images/tequila-logo.png';
+import graphqlLogo from '../images/graphql-logo.png';
+import airport from '../images/airport.jpg';
+
+export default () => (
+  <Headline>
+    <HeadlineTextContainer>
+      <Heading type="display">
+        Ever dreamt of building the next Kiwi.com?
+      </Heading>
+    </HeadlineTextContainer>
+    <Container>
+      <ContainerTop>
+        <HeaderAndDesignWrapper>
+          <BackgroundGradientAccent>
+            <GreenRectangle />
+          </BackgroundGradientAccent>
+          <Heading type="title2">
+            You've had a look at Tequila, but you've always preffered your
+            Tequila mixed with something else. Just grab a Margarita.
+          </Heading>
+        </HeaderAndDesignWrapper>
+        <ContainerIcons>
+          <img src={kiwiLogo} height={43} alt="kiwi_logo" />
+          <img src={reactLogo} height={45} alt="react_logo" />
+          <img src={tequilaLogo} height={43} alt="tequila_logo" />
+          <img src={graphqlLogo} height={43} alt="graphql_logo" />
+        </ContainerIcons>
+      </ContainerTop>
+      <ContainerBottom>
+        <ContainerLeftText>
+          <Heading>Get started</Heading>
+          <WrapperText>
+            <Text size="large">
+              PLACEHOLDER!!!!!You've had a look at Tequila, but you've preffered
+              yourTequila Tequila mixed with something else. Just grab
+              Margarita.You've had a look at Tequila, but you've always
+              preffered your Tequila mixed with something else.
+            </Text>
+          </WrapperText>
+          <WrapperButtons>
+            <Button>Get started</Button>
+            <WrapperRightButton>
+              <Button type="white">Link to Github</Button>
```

Comment:
Shouldn't be these `<Button>Get started</Button>` and ` <Button type="white">Link to Github</Button>` links?

Reply:
Totally missed that. Thanks! Added links to margarita and github